app/routes/companyInterest/CompanyInterestListRoute.js

```diff
@@ -6,34 +6,65 @@ import {
   deleteCompanyInterest,
   fetch
 } from 'app/actions/CompanyInterestActions';
+import { fetchSemesters } from 'app/actions/CompanyActions';
 import CompanyInterestList from './components/CompanyInterestList';
 import { selectCompanyInterestList } from 'app/reducers/companyInterest';
+import { selectCompanySemesters } from 'app/reducers/companySemesters';
 import { LoginPage } from 'app/components/LoginForm';
 import replaceUnlessLoggedIn from 'app/utils/replaceUnlessLoggedIn';
 import prepare from 'app/utils/prepare';
+import { push } from 'react-router-redux';
+import { semesterToText } from './utils';
+import type { CompanySemesterEntity } from 'app/reducers/companySemesters';
 
-const loadCompanyInterests = (props, dispatch) => dispatch(fetchAll());
+const loadData = ({ params }, dispatch) =>
+  Promise.all([
+    dispatch(fetchAll()).catch(),
```

Comment:
Is there a reason for this empty `catch`? If the action fails, it will not report it to sentry